File: CS1/Windows/Form_ctrl1/Form_ctrl1/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace Form_ctrl1
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e)
        {
          //  button1.Click += button1_Click;
        }

        void button1_Click(object sender, EventArgs e)
        {
            
        }

        //private void button1_Click(object sender, EventArgs e)
        //{
        //    //printDialog1.ShowDialog();
        //    MessageBox.Show( "Welcome");
        // }

        private void button1_Click_1(object sender, EventArgs e)
        {
            String name = textBox1.Text;
            String Course = comboBox1.SelectedItem.ToString();
            String gender="";
            if(radioButton1.Checked)
                gender="Male";
            else if(radioButton2.Checked)
                gender = "Female";
            String lan="";
            if (checkBox1.Checked)
                lan += checkBox1.Text+"   ";
            if (checkBox2.Checked)
                lan += checkBox2.Text+"   ";
            if (checkBox3.Checked)
                lan += checkBox3.Text+"   ";

            listBox1.Items.Add(name+"   "+Course+"  "+ gender +"  "+lan);
        }

        private void comboBox1_SelectedIndexChanged(object sender, EventArgs e)
        {
            label6.Text = comboBox1.SelectedIndex.ToString();
        }

        private void button2_Click(object sender, EventArgs e)
        {
            int i=listBox1.Items.Count;
            listBox1.Items.RemoveAt(i-1);
            
        }
    }
}

File: VB.NET/windows/radio_check/radio_check/Form1.vb
Public Class Form1
    Dim a As String




    Private Sub Button1_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button1.Click
        If (RadioButton1.Checked) Then
            a = RadioButton1.Text
        ElseIf (RadioButton2.Checked) Then
            a = RadioButton2.Text
        End If

        If (CheckBox1.Checked) Then
            a += vbNewLine + CheckBox1.Text
        End If
        If (CheckBox2.Checked) Then
            a += vbNewLine + CheckBox2.Text
        End If
        If (CheckBox3.Checked) Then
            a += vbNewLine + CheckBox3.Text
        End If


        TextBox1.Text = a
    End Sub

    Private Sub Form1_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load

    End Sub
End Class
